annotations/build.gradle.kts

```diff
@@ -47,10 +49,9 @@ testing {
           implementation("org.springframework.boot:spring-boot-starter-webflux")
           implementation("org.springframework.boot:spring-boot-starter-data-jpa")
           implementation("org.springframework.boot:spring-boot-testcontainers")
-          implementation(platform("org.testcontainers:testcontainers-bom:1.19.6"))
```

Comment:
This configuration has no effect, as you are using the [Spring Boot Dependency Management Plugin for Gradle](https://docs.spring.io/dependency-management-plugin/docs/current/reference/html/).